src/Microsoft.DotNet.Build.Tasks/FilterRuntimesFromSupports.cs

```diff
@@ -57,7 +59,7 @@ public static JObject GenerateCustomSupportsClause(List<TfmRidPair> applicableTf
             var tfmRidsForSupports = new JObject();
             foreach (var tfmRidPair in applicableTfmRidPairs)
             {
-                tfmRidsForSupports[tfmRidPair.framework] = new JArray(tfmRidPair.runtimeIdentifier);
+                tfmRidsForSupports[NuGetFramework.Parse(tfmRidPair.framework).GetShortFolderName()] = new JArray(tfmRidPair.runtimeIdentifier);
```

Comment:
I left this change in for consistency when people want to debug generated project.json so that they can determine what changed and not have to look up tfm short form -> long form mappings.